.zenodo.json

```diff
@@ -7,6 +7,11 @@
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
       "orcid": "0000-0003-3396-6154"
     },
+    {
+      "name": "Bastrakova, Kseniia",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
```

Comment:
@KseniaBastrakova You contributed in June 2022. Were you still at HZDR back then or do you require a different affiliation here?

Reply:
Answer: HZDR is correct.